lib/vernier/output/top.rb

```diff
@@ -3,14 +3,16 @@
 module Vernier
   module Output
     class Top
-      def initialize(profile)
+      def initialize(profile, row_limit)
```

Comment:
Instead of requiring this arg, we could move default to 20 here instead of [during initialization](https://github.com/jhawthorn/vernier/blob/f43167453e7606bafc603e3cd57036cacb22cdd5/exe/vernier#L110). I can address this in a follow-up though.